releases/release-1.19/release_team.md

```diff
@@ -0,0 +1,16 @@
+# Kubernetes 1.19 Release Team
+
+| **Role**         | **Name** (**GitHub / Slack ID**)                                                                                                                        | **Shadow Name(s) (GitHub / Slack ID)**                                                                                              |
+| ---------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
```suggestion
|---|---|---|
```

Prefer just three dashes here, since the column width will shift every time you add someone.